### PR TITLE
enable speech_recognition and recommedation new api examples for pytorch 

### PR DESCRIPTION
**Pull request by changwangss**
## Type of Change

speech recognition (change torchaudio eager+dynamic to fx+static)and recommedation.

## Description
enable examples for recommendation, speech recognition

detail description 
JIRA ticket: [ILITV-2466](https://jira.devtools.intel.com/browse/ILITV-2466)

## Expected Behavior & Potential Risk

the expected behavior that triggered by this PR 

## How has this PR been tested?

how to reproduce the test (including hardware information)

## Dependency Change?

any library dependency introduced or removed

### CODE REVIEW

File: examples/pytorch/recommendation/dlrm/quantization/ptq/fx/run_benchmark.sh [hunk 1]
```diff
@@ -52,7 +52,7 @@ function init_params {
 # run_benchmark
 function run_benchmark {
     if [[ ${mode} == "accuracy" ]]; then
-        mode_cmd=" --benchmark"
+        mode_cmd=" --accuracy_only"
         default_bs=16384
     elif [[ ${mode} == "benchmark" ]]; then
```
Comment (chensuyue) [inline examples/pytorch/recommendation/dlrm/quantization/ptq/fx/run_benchmark.sh:57]: image
please use accuracy/performance for benchmark mode.

File: examples/pytorch/speech_recognition/torchaudio_models/quantization/ptq_static/fx/run_tuning.sh [hunk 2]
```diff
@@ -44,9 +44,16 @@ function init_params {
 # run_tuning
 function run_tuning {
     extra_cmd=""
-    if [ -n "$input_model"];then
-      input_model=$topology
```
Comment (chensuyue) [inline examples/pytorch/speech_recognition/torchaudio_models/quantization/ptq_static/fx/run_tuning.sh:48]: No need this one, test code will pass input model as you set in json file.

**Comment by chensuyue** [comment]
[extension test](https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/3795/)

**Comment by chensuyue** [comment]
retest after bug fix: extension [test](https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/3811/)

**Review by PenghuiCheng — APPROVED** [review]

**Comment by changwangss** [comment]
torchaudio: hubert_fx: https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/3837/

**Comment by chensuyue** [comment]
https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/3839/artifact/report.html

**Comment by changwangss** [comment]
https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/3871/

**Comment by changwangss** [comment]
https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/3874/

**Comment by changwangss** [comment]
https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/3885/

**Pull request merged**